lib/App.js

```diff
@@ -1833,20 +1826,164 @@ class App {
             return true;
           }
         }
-      ]);
+      ]); 
 
-      // Create a custom object to inject en flag for the placeholder.
-      flowArguments.push({
-        type: argumentType,
-        name: argumentName,
-        placeholder: { en: argumentPlaceholder },
-      });
+      if (argumentType != "dropdown" && argumentType != "color" && argumentType != "droptoken" && argumentType != "range"){
```

Comment:
Prefer whitelist over blacklist, it's more future-proof.

```js
['text', 'autocomplete', ...].includes(argumentType)
```

Reply:
Also use single quotes.

Reply:
I think using a switch here is nice, with the small downside that you may need to copy the placeholder question for number.

```js
switch (argumentType) {
  case 'number':
    // question placeholder
    // question min/max/step
    break;

  case 'range':
    // question min/max/step
    break;

  case 'dropdown':
    // question values
    break;

  case 'text':
  case 'autocomplete':
  case 'date':
  case 'time':
  case 'device':
  default:
    // question placeholder
    break;
}